ecs-cli/modules/config/params_test.go

```diff
@@ -156,6 +162,50 @@ func TestNewCliParamsWhenPrefixKeysAreNotPresent(t *testing.T) {
 	assert.Equal(t, ecscli.CFNStackNamePrefixDefaultValue+clusterName, params.CFNStackName, "Expected CFNStackName to be default")
 }
 
+func TestNewCliParamsWithAWSProfile(t *testing.T) {
+	// Keys in env vars take highest precedence; ensure they are not set
+	os.Unsetenv("AWS_ACCESS_KEY")
+	os.Unsetenv("AWS_SECRET_KEY")
+
+	configContents := `[awsprofile]
```

Comment:
super nitpicky and asking mostly out of curiosity, but is there a way to interpolate `awsProfileName` here?

Reply:
There are other ways that one can do this, but they end up making the code harder to read. The way I did it shows very clearly what the dummy file looks like. Here's the best alternate solution:
```
configContents := "[" + awsProfileName + "]" +
		"aws_access_key_id = aws-access" +
		"aws_secret_access_key = aws-secret"
```

This compiles but I don't really like it either. It also causes the test case to fail for some reason, so I'm going to stick to the old string.

Reply:
Oh, it causes the test case to fail because I forgot to add newlines. Still, I don't think like the way this code reads, so I'm going to keep the old code.